research/CongressKeynote/Congress-Keynote.md

```diff
@@ -0,0 +1,466 @@
+
+{% include figure.html src="Slides/Slide8.png" caption="Slide 8." %}
+
+ What *is* important to note in the above spectrograms is that each of the indices provides a different view of the four-hour soundscape. Acoustic indices may be compared to camera lens filters -- different filters offer different views of a landscape.  
+
+---
+## Slide 9: False-colour spectrograms
+{% include figure.html src="Slides/Slide9.png" caption="Slide 9." %}
+
+The next step in the production of long-duration spectrograms was inspired by false-colour satellite imagery in which pictures of the earth’s surface are rendered in three colours by assigning the red, green and blue (RGB) colours to sensors that respond to different parts of the electromagnetic spectrum. In this case we assign RGB to the three different spectrograms and thereby produce the long-duration, false-colour (LDFC) spectrogram seen on the right-hand side of this slide. Note how much detailed structure in the sound-space is revealed. 
+
+---
+
+## Slide 10: The utility of 24-hour, false-colour spectrograms
+{% include figure.html src="Slides/Slide10.png" caption="Slide 10." %}
+
+Here we see how much additional acoustic structure can be seen in a false-colour spectrogram. Each of the three spectrograms is of the same 24-hour recording made in open eucalypt woodland at the [Samford Ecological Research Facility](https://www.qut.edu.au/institute-for-future-environments/facilities/samford-ecological-research-facility) on the outskirts of Brisbane city. (This is the same recording from which the four hours of the previous slide were extracted.)
+
+Each spectrogram starts at midnight and finishes the following midnight. Midday is in the centre. The top spectrogram is the one produced by Audacity. The middle spectrogram is the false-colour one produced by assigning the ACI, H(t) and CVR indices to the RGB channels respectively. The basic structure of the forest soundscape is easily interpreted. The morning chorus is clearly visible at 4:45am. It tends towards white in colour because all indices have high values. The onset of evening is indicated by the appearnace of acoustic tracks due to insects. But clearly there are many other events. The colours of acoustic events in the spectrogram depend both on the three indices used and on the characteristic distribution of acoustic energy in the animal call(s) that contribute to the event. With some experience you begin to recognise the different sounds of interest.
+
+For convenience, we refer to these spectrograms as long-duration, false-colour spectrograms or LDFC spectrograms.
+
+Different combinations of indices give different views of the soundscape. Note how less acoustic structure is visible in the bottom spectrogram because two of the three indices are correlated. So here is an important point - the three indices used to construct an LDFC spectrogram should be minimally correlated. (For more detail see previous tutorials and publications.)
```

Comment:
Do you want to add a sentence explaining which three indices are used in the bottom image?